### DISCUSSION TITLE
0x4675128107706e2a9b3fba11abbb11e0903375c0c750d379c0d39e84b6bd64ef

### DISCUSSION THREAD

**Discussion by nguyentienddddung** [zkSync-Community-Hub/zksync-developers#461] [Contract Verification]
### Environment

Mainnet

### Issue Type

- [ ] Plugin
- [X] Block Explorer

### Contract Address

0xF2F3B8FfC308f341b8D882A613B2fa9c00D45671

### Compiler Type

Single file

### zkSolc Version

2

### Solc Version

2

### Contract Name

3

### Contract Code

4

### Constructor Arguments

3

**Comment by bxpana** [comment]
@nguyentienddddung looks like you didn't properly fill out the information. Did you have a contract verification issue?
